docs/spatial-audio-rfc.md

```diff
@@ -59,10 +61,12 @@ aligned(8) class SpatialAudioBox extends Box(‘SA3D’) {
 
 - `channel_map` is a sequence of 32-bit unsigned integers that maps audio channels in a given audio track to ambisonic components, given the defined `ambisonic_channel_ordering`. The sequence of `channel_map` values should match the channel sequence within the given audio track.
 
-  For example, consider a 4-channel audio track containing ambisonic components *W*, *X*, *Y*, *Z* at channel indexes 0, 1, 2, 3, respectively. For `ambisonic_channel_ordering = 0` (ACN), the ordering of components should be *W*, *Y*, *Z*, *X*, so the `channel_map` sequence should be `0`, `2`, `3`, `1`.
+  For the example case of ambisonic_type == 'Periphonic', consider a 4-channel audio track containing ambisonic components *W*, *X*, *Y*, *Z* at channel indexes 0, 1, 2, 3, respectively. For `ambisonic_channel_ordering = 0` (ACN), the ordering of components should be *W*, *Y*, *Z*, *X*, so the `channel_map` sequence should be `0`, `2`, `3`, `1`.
 
   As a simpler example, for a 4-channel audio track containing ambisonic components *W*, *Y*, *Z*, *X* at channel indexes 0, 1, 2, 3, respectively, the `channel_map` sequence should be specified as `0`, `1`, `2`, `3` when `ambisonic_channel_ordering = 0` (ACN).
 
+  For the example case of ambisonic_type == 'Periphonic' with the the non_diegetic flag set, the stored audio will consist of 4 ambisonic components W, Y, Z, X and non-diegetic stereo components L and R. In this case, the SA3D atom will define num_channels = 6 and a channel_map of 0,1,2,3,4,5 indicating that the channels are laid out in the file as W, Y, Z, X, L, R. This representation extends to different layouts of ambisonics and non-diegetic components. For example, a channel layout of 2, 3, 4, 5, 0, 1  indicates that the layout of the stored audio is L, R, W, Y, Z, X.
```

Comment:
Format nit: it looks like ambisonic components are italicized (*W* vs. W), fields and values are in code type, and that there should be spacing between your comma-delimited channel_map specification.

`ambisonic_type = 0` (Periphonic)
`non_diegetic = 1`
*W*, etc...
`num_channels = 6`
`channel_map` specified as `0`, `1`, etc...

Also I'm confused by the second "2, 3, 4, 5, 0, 1" example -- if the expected order is WYZXLR, would 234501 map to ZXLRWY?

Reply:
Updated font types. Also, you are absolutely correct about that last channel map example. For components in the order L,R,W,Y,Z,X, the channel map sequence should be 4,5,0,1,2,3. I've correct the example. Thanks for catching.